### DISCUSSION TITLE
`FormErrors`: Property "hasErrors" was accessed during render but is not defined on instance.

### DISCUSSION THREAD

**Discussion by letmaik** [vueform/vueform#217] [Questions]
### Environment

    "@vueform/vueform": "^1.9.4",
    "vue": "^3.4.21"
    "@vitejs/plugin-vue": "^5.0.4",
    "postcss": "^8.4.38",
    "tailwindcss": "^3.4.3",
    "vite": "^5.2.0"

### Reproduction

```vue
    <Vueform>
      <template #empty>
        <FormErrors v-if="hasErrors" />
        ...
```

### Describe the bug

```
App.vue:185  [Vue warn]: Property "hasErrors" was accessed during render but is not defined on instance. 
  at <Vueform> 
  at <App>
```

### Additional context

I'm trying to follow https://vueform.com/reference/form-errors#basic-usage.

**Comment by letmaik** [comment]
Note that I'm using `<template #empty>` because of https://github.com/vueform/builder/issues/56.

**Comment by adamberecz** [comment]
It's not really an issue, though it would make sense to export these variables in `#empty` scope (note for self).

Nevertheless, here's how you can solve it in the meantime:
```html
<template>
  <Vueform ref="form$">
    <template #empty>
      <FormErrors v-if="form$?.hasErrors" />
    </template>
  </Vueform>
</template>

<script setup>
import { ref } from 'vue'

const form$ = ref(null)
</script>
```

**Reply by letmaik** [reply]
Thanks, that works! And I do understand why now.
